README.md

```diff
@@ -88,31 +107,23 @@ Fetches details about a single market. Example response:
 }
 ```
 
-### `veil.getOrders(market: Market, options?: { page: number })`
+### `veil.getBids(market: Market, tokenType: "long" | "short")` and `veil.getAsks(market: Market, tokenType: "long" | "short")`
 
-Fetches the open orders in a market. Example response:
+The `getBids` and `getAsks` methods let you fetch the orderbook for a market. You can fetch orders for either LONG or SHORT tokens by passing `tokenType` as a second argument (in Veil markets, the LONG and SHORT orderbooks are always mirror images of each other).
+
+Bids are sorted by price descending, and asks are sorted by price ascending, so you can get the spread of a market by comparing the first bid and first ask.
+
+Example response:
 ```js
 {
   results: [  
     {  
-      uid: "3e4fd40d-176f-432f-8f0b-d0a600d55a1f",
-      status: "open",
-      createdAt: 1543509213537,
-      expiresAt: 100000000000000,
-      type: "limit",
-      tokenType: "short",
-      side: "buy",
-      longSide: "sell",
-      longPrice: "7707",
-      shortPrice: "2293",
-      token: "0x598b46d68e3e03f810a45d7d8dc9af5afdbafd56",
-      tokenAmount: "100000000000000",
-      "tokenAmountFilled":"0",
-      currency: "0xe7a67a41b4d41b60e0efb60363df163e3cb6278f",
-      currencyAmount: "229300000000000000",
-      currencyAmountFilled: "0",
-      postOnly: false,
-      market: null
+      price: "7000",
+      tokenAmount: "100000000000000"
```

Comment:
Not worth doing something like this is it?
```
{ price: {raw: "7000", float: 0.7}, tokenAmount: {raw: "100000000000000", float: 1.0} }
```

Reply:
Maybe worth eventually adding convenience fields like `tokenAmountFloat` to 1) keep it backward compatible and 2) allow people to ignore it